packages/apps/shopify-app-remix/src/react/components/AppProvider/AppProvider.tsx

```diff
@@ -5,7 +5,7 @@ import {
 // This leads to some TS errors, but it does compile to something that works
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-ignore
```

Comment:
Is this @ts-ignore still needed after this change?

Reply:
Good call, it should not be. Removing.